src/dune_rules/lib_info.mli

```diff
@@ -150,6 +150,8 @@ val special_builtin_support : _ t -> Special_builtin_support.t option
 
 val modes : _ t -> Lib_mode.Map.Set.t
 
+val modules : _ t -> Modules.t Source.t
```

Comment:
Hm, this type isn't quite correct actually. it should really be:

```
type t =
  | Local
  | Unavailable
  | Present of Modules.t
```

Since findlib modules do not have this information.